tools/cldr-apps/src/main/webapp/js/survey.js

```diff
@@ -2155,32 +2155,49 @@ require(["dojo/ready"], function(ready) {
 			if (tr) {
 				var theRow = tr.theRow;
 				// this also marks this row as a 'help parent'
-				theHelp = createChunk("", "div", "alert alert-info fix-popover-help vote-help");
 
 				if (theRow.xpstrid) {
-					var deferHelpSpan = document.createElement('span');
-					theHelp.appendChild(deferHelpSpan);
+                                        theHelp = createChunk("", "div", "alert alert-info fix-popover-help vote-help");
+
+                                        function showHelp(theHelp, data) {
+                                            theHelp = $(theHelp);
```

Comment:
It would be great to avoid a new nested function here. The enclosing anonymous function is too large already, over 200 lines. New js functions should go in strict js files. Maybe a new file CldrStHelp.js, with an IIFE pattern?

Also the indentation is messed up, and that's partly a consequence of survey.js already being so messed up -- another reason to minimize the code added to it

Reply:
@btangmu sounds good